lottie-ios/Classes/Private/LOTAnimationView.m

```diff
@@ -310,7 +310,9 @@ - (void)playFromFrame:(nonnull NSNumber *)fromStartFrame
   }
   _animationProgress = [self _progressForFrame:currentFrame];
   
-  NSTimeInterval offset = MAX(0, (_animationProgress * (_sceneModel.endFrame.floatValue - _sceneModel.startFrame.floatValue)) - fromStartFrame.floatValue) / _sceneModel.framerate.floatValue;
+  // If the animation is playing backrwars, to get the offset we need to invert the progress.
+  CGFloat offsetProgress = playingForward ? _animationProgress : 1 - _animationProgress;
```

Comment:
This should only be the case if fromFrame is less than toFrame.
If speed is negative then the offset shouldnt make a difference?

Reply:
I was finding the offset was causing the animation to jump to the finish before playing. Changing the offset here seems to have fixed it, but maybe it's masking another problem?